awscli/customizations/awslambda.py

```diff
@@ -54,6 +57,11 @@ def _extract_code_and_zip_file_arguments(session, argument_table, **kwargs):
     )
 
 
+def _modify_zipfile_docstring(session, argument_table, **kwargs):
+    if 'zip-file' in argument_table:
+        argument_table['zip-file'].argument_model.documentation = ZIP_DOCSTRING
```

Comment:
I would prefer to treat the argument_model as read_only.  I don't think there's anywhere else in the code where we mutate a model.  What about making the `documentation` property settable on an argument object?